Source/swiftlint/Commands/Analyze.swift

```diff
@@ -13,7 +13,7 @@ extension SwiftLint {
         var quiet = false
         @Option(help: "The path of the full xcodebuild log to use when running AnalyzerRules.")
         var compilerLogPath: String?
-        @Option(help: "The path of a compilation database to use when running AnalyzerRules.")
+        @Option(help: "The path of a compilation database to use when running AnalyzerRules")
```

Comment:
Why this change? Don't most help descriptions end with a period?

Reply:
this was just to trigger sos-check, otherwise it'd skip because both binaries would have the same signature. I'll revert it before merging

Reply:
Should work again now that you've fixed `OSSCheck` to run when `tools/oss-check` changes.

Reply:
it didn't because of this check: https://github.com/realm/SwiftLint/blob/main/tools/oss-check#L324-L329

Reply:
Ah right.